crates/macro/src/html_tree/html_tag/mod.rs

```diff
@@ -121,13 +129,6 @@ impl ToTokens for HtmlTag {
         let set_checked = checked.iter().map(|checked| {
             quote_spanned! {checked.span()=> #vtag.set_checked(#checked); }
         });
-        let add_disabled = disabled.iter().map(|disabled| {
-            quote_spanned! {disabled.span()=>
-                if #disabled {
-                    #vtag.add_attribute("disabled", &"true");
-                }
-            }
-        });
         let add_selected = selected.iter().map(|selected| {
```

Comment:
We don't need special handling for `selected` with your change. Can you please remove the special handling and ensure that `selected` works properly as a boolean attribute?